jme3-core/src/main/java/com/jme3/shadow/AbstractShadowFilter.java

```diff
@@ -48,35 +44,34 @@
 import com.jme3.util.clone.Cloner;
 import com.jme3.util.clone.JmeCloneable;
 
-import java.io.IOException;
-
 /**
  * Generic abstract filter that holds common implementations for the different
  * shadow filters
  *
- * @author Rémy Bouquet aka Nehon
+ * @author Nehon
  */
-public abstract class AbstractShadowFilter<T extends AbstractShadowRenderer> extends Filter implements Cloneable, JmeCloneable {
+public abstract class AbstractShadowFilter<T extends AbstractShadowRenderer> extends Filter implements JmeCloneable {
```

Comment:
Why remove Cloneable? (Why was it here in the first place?)

Reply:
The `JmeCloneable` interface already extends` java.lang.Cloneable`. Therefore, the explicit extension is redundant and its removal simplifies the code without altering its behavior. Therefore, what checks have you already performed that led you to raise this question?

Reply:
Ah yes good point. No objection to removing it then